src/agent/rustjail/src/container.rs

```diff
@@ -605,6 +607,13 @@ fn do_init_child(cwfd: RawFd) -> Result<()> {
         env::set_var(v[0], v[1]);
     }
 
+    // set the "HOME" env getting from "/etc/passwd"
+    if env::var_os(HOME_ENV_KEY).is_none() {
```

Comment:
Could you add some unit tests for this please?

Reply:
Hi @jodh-intel 

Since the home env setting was done in the container process init function, which has a lot of communication with parent process, it's not easy to setup a unit test for this big function, but I can add a test case in CI to verify the env setting in containers. Does that work for you?